### PR TITLE
Added new opportunity

### PR DESCRIPTION
**Pull request by anjalisingh23**
Added a new opportunity and corrected overflow

### CODE REVIEW

File: css_final.css [hunk 1]
```diff
@@ -1,112 +1,96 @@
...
-    border-radius: 4px;
-    box-shadow: 0px 0px 3px rgba(0, 0, 0, 0.9);
-    max-width: 90%;
-    max-height: fit-content;
-
-    
+  margin-top: 40px;
+  margin-left: 20px;
+  margin-right: 20px;
+  padding-bottom: 5%;
```
Comment (souvikrajsingh) [inline css_final.css]: your code is good to go Anjali @anjalisingh23 just remove this padding here

Comment (anjalisingh23) [reply css_final.css]: I added padding cuz adding new row was causing overflow
Shall I still remove it?

Comment (souvikrajsingh) [reply css_final.css]: yes ! I love your PR but if you are adding only a opportunity you should just only edit the index.html file Anjali! 

i would edit only the index.html file and submit a new PR 

Tag me if you need any assitance !

Comment (anjalisingh23) [reply css_final.css]: @souvikrajsingh  I commited css file but how to submit a new PR?

Comment (souvikrajsingh) [reply css_final.css]: do it from the beginning anjali, delete the local copy you have of this repo on your system and fork and clone it again.

Work ONLY on the index.html file.

Just add the name of the opportunity as directed and you are good to go! 
Cheers

Comment (anjalisingh23) [reply css_final.css]: okh got it
Thankyou

**Pull request closed without merging**